packages/dd-trace/test/custom-metrics.spec.js

```diff
@@ -53,7 +53,7 @@ describe('Custom Metrics', () => {
       if (stdout) console.log(stdout)
       if (stderr) console.error(stderr)
 
-      expect(metricsData.split('#')[0]).to.equal('page.views.data:1|c|')
+      expect(metricsData.split('#')[0]).to.equal('page.views.data:1|g|')
```

Comment:
Does this mean all increments switched from a count metric to a gauge metric? 

```js
tracer.dogstatsd.increment('page.views.data')
```

Reply:
Similar to runtime metrics, instead of sending the increment to the agent/backend and having them handle actually applying it, it is now applied upfront.